myhpi/core/models.py

```diff
@@ -92,10 +84,9 @@ def get_visible_minutes(self, request):
         minutes_ids = self.get_children().exact_type(Minutes).values_list("id", flat=True)
         user_groups = get_user_groups(request.user)
         minutes_list = (
-            Minutes.objects.filter(id__in=minutes_ids, group=self.group)
+            Minutes.objects.filter(id__in=minutes_ids)
             .filter(Q(visible_for__in=user_groups) | Q(is_public=True))
```

Comment:
wouldn't it be possible to apply this filter to self.get_children().exact_type(Minutes) instead?

Reply:
Don't know why this is this way

Reply:
But it doesn't work the other way, so I'm just going to merge